server/src/main/java/org/candlepin/controller/Entitler.java

```diff
@@ -235,10 +235,15 @@ public void adjustEntitlementQuantity(Consumer consumer, Entitlement ent,
                     "Auto-attach is disabled for owner \"{0}\"{1}.",
                     owner.getKey(), hypMessage));
             }
-            else {
+            else if (owner.isAutobindDisabled()) {
                 throw new AutobindDisabledForOwnerException(i18n.tr(
                     "Auto-attach is disabled for owner \"{0}\"{1}.",
-                    owner.getKey(), caMessage));
+                    owner.getKey()));
```

Comment:
The `caMessage` variable was accidentally removed here

Reply:
This was removed because in this `if` condition the `caMessage` is empty.